File: modules/layout/source/types/layout_types/project.ts
import { Selector } from './selector'
import { Tag } from './tag'
import { Func } from './func'

// Описание проекта
export class Project {
    id: string // ай-ди элемента
    name: string // Название проекта
    description: string // Описание проекта (данного)
    title: string //  title
    favicon: string // фавиконка
    selectors: Selector[] // css
    tags: Tag[] // html
    functions: Func[] // js
}

File: modules/layout/app/types/layout_types/project.go
package layout_types

// Описание проекта
type Project  struct{
    Id string // ай-ди элемента
    Name string // Название проекта
    Description string // Описание проекта (данного)
    Title string //  title
    Favicon string // фавиконка
    Selectors []Selector // css
    Tags []Tag // html
    Functions []Func // js
}
